README.md

```diff
@@ -10,13 +10,14 @@ Discord bot for shooting_the_breeze.java
 ## Testing with you own bot
 1. [Setting up a bot application](https://discordjs.guide/preparations/setting-up-a-bot-application.html)
 2. [Adding your bot to servers](https://discordjs.guide/preparations/adding-your-bot-to-servers.html)
-3. Setup a config.json as shown (you'll need to enable dev settings to get the client and guild id through the right-click in discord):
+3. Setup a config.json as shown in the src/ folder (you'll need to enable dev settings to get the client and guild id through the right-click in discord):
 ```
 {
 	"clientId": "your-bots-id",
 	"guildId": "your-guild-id",
 	"token": "your-token-goes-here"
 }
 ```
```

Comment:
Is there an install step that isn't captured in the readme?

Reply:
Yeah but I'm too lazy to do that in this pr I'll make a separate issue